src/backend/impl/src/controllers/dev_controller.rs

```diff
@@ -0,0 +1,31 @@
+use crate::{
```

Comment:
I'm not sure about this... Having a `DevController` and `DevService` feels weird, but it means that any "dev only" endpoints we might have are all in one place. If we put them in the relevant controllers then we'd have a lot more `#[cfg(feature = "dev")]` scattered around and it could be easy to miss one.

Reply:
> it means that any "dev only" endpoints we might have are all in one place

Makes sense to me!